src/index.ts

```diff
@@ -1,7 +1,16 @@
 import {Agenda} from './agenda';
+import {Contact} from './agenda';
 
 let agenda = new Agenda();
 
-agenda.addContact('Vesal', 'Fayazi', 'Hagsätra');
+let contact = new Contact('Vesal', 'Fayazi', 'Regeringsgatan 28');
+let contact1 = new Contact('Miguel', 'Dominguez', 'Spanish street 5');
+let contact2 = new Contact('Poop', 'Hat', 'Diarrhea alley 24');
 
-console.log(agenda);
\ No newline at end of file
+agenda.addContact(contact);
+agenda.addContact(contact1);
+agenda.addContact(contact2);
+
+let searchContact = agenda.searchContactBySurname('Hat');
+
+console.log(searchContact);
```

Comment:
I would like to have another if statement in the method, but I'm getting a error message that return types are not the same. So I have to try to find another solution...